### DISCUSSION TITLE
Can we add specifc labels purely to the onboarding MRs?

### DISCUSSION THREAD

**Discussion by KevinM2k** [renovatebot/renovate#21840] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

The RENOVATE_ADD_LABELS env var is great to add labels to the MR's, however is there a way of adding specific ones to the onboarding MR's, so I can add a 'onboarding' label to only those?

**Comment by rarkins** [comment]
Looks like we don't have that support today, but it sounds like a good idea

**Reply by HonkingGoose** [reply]
I created this issue for us:

- #21890
